core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -128,6 +129,15 @@ case class GroupSummary(state: String,
                         members: List[MemberSummary])
 
 /**
+ * Case class used to represent group metadata (including the group coordinator) for the DescribeGroup API
+ */
+case class ConsumerGroupSummary(state: String,
+                                protocolType: String,
```

Comment:
Upon further consideration, I think I'm going to adopt your first suggestion, because I need the current `describeGroup` method to return the coordinator as part of its output; so I can report the coordinator's broker id. I'm going to use the method name `getConsumerGroupSummary` instead.